tests/cicd.etest

```diff
@@ -0,0 +1,341 @@
+    assert_eq "etest"                               "${repo_slug}"
+    assert_eq "0.0"                                 "${series}"
+    assert_eq "0.0.0.0"                             "${version}"
+    assert_eq "v0.0.0.0"                            "${version_tag}"
+    assert_eq "v0.0.0.1"                            "${version_tag_next}"
+}
+
+ETEST_cicd_info_offset()
+{
+    etestmsg "Adding another commit"
+    echo "foo" > "file1.txt"
+    git add "file1.txt"
+    git status
+    git commit -am "Bump file1.txt"
+
+    # Create CICD Pack
+    etestmsg "Test CI/CD Info"
+    cicd_info info
+    $(pack_import info)
+    cicd_print --json info
+
+    etestmsg "Validating CI/CD Info"
+    assert_eq "v1.0.0.0"                            "${base_tag}"
+    assert_eq "develop"                             "${branch}"
+    assert_eq "0"                                   "${build}"
+    assert_eq "$(git rev-parse HEAD)"               "${commit}"
+    assert_eq "$(git rev-parse HEAD | cut -c1-10)"  "${commit_short}"
+    assert_eq "1"                                   "${major}"
+    assert_eq "0"                                   "${minor}"
+    assert_eq "1"                                   "${offset}"
+    assert_eq "git@github.com:elibs/etest.git"      "${origin_url}"
+    assert_eq "0"                                   "${patch}"
+    assert_eq "etest"                               "${repo_slug}"
+    assert_eq "1.0"                                 "${series}"
+    assert_eq "1.0.0.0-1-g${commit_short}"          "${version}"
+    assert_eq "v1.0.0.0-1-g${commit_short}"         "${version_tag}"
+    assert_eq "v1.0.0.1"                            "${version_tag_next}"
+}
+
+ETEST_cicd_info_print()
+{
+    # Create CICD Pack
+    etestmsg "Test CI/CD Info"
+    cicd_info info
+    $(pack_import info)
+    cicd_print info > output
+
+	cat > expect <<-EOF
+	base_tag="v1.0.0.0"
+	branch="develop"
+	build="0"
+	commit="${commit}"
+	commit_short="${commit_short}"
+	major="1"
+	minor="0"
+	offset="0"
+	origin_url="git@github.com:elibs/etest.git"
+	patch="0"
+	repo_slug="etest"
+	series="1.0"
+	version="1.0.0.0"
+	version_tag="v1.0.0.0"
+	version_tag_next="v1.0.0.1"
+	EOF
+
+    einfo "Expect"
+    cat expect
+
+    einfo "Actual"
+    cat output
+
+    diff -u expect output
+}
+
+ETEST_cicd_info_print_json()
+{
+    # Create CICD Pack
+    etestmsg "Test CI/CD Info"
+    cicd_info info
+    $(pack_import info)
+    cicd_print --json info > output.json
+
+	cat > expect.json <<-EOF
+	{
+	  "base_tag": "v1.0.0.0",
+	  "branch": "develop",
+	  "build": "0",
+	  "commit": "${commit}",
+	  "commit_short": "${commit_short}",
+	  "major": "1",
+	  "minor": "0",
+	  "offset": "0",
+	  "origin_url": "git@github.com:elibs/etest.git",
+	  "patch": "0",
+	  "repo_slug": "etest",
+	  "series": "1.0",
+	  "version": "1.0.0.0",
+	  "version_tag": "v1.0.0.0",
+	  "version_tag_next": "v1.0.0.1"
+	}
+	EOF
+
+    einfo "Expect"
+    #jq . expect.json
```

Comment:
Did you mean to leave this commented out?

Reply:
I did not. I'll fix that... but don't want to re-do 3 hours of tests, so I'll merge it and then fix it.